collect/cache/traceSentCache.go

```diff
@@ -11,7 +11,13 @@ type TraceSentRecord interface {
 	// Rate() returns the sample rate for the trace
 	Rate() uint
 	// DescendantCount returns the count of items associated with the trace, including all types of children like span links and span events.
-	DescendantCount() uint
+	DescendantCount() uint32
```

Comment:
Can we plese make the int32 an implementation detail of the record, and have all these functions return a normal int or an int64?